### PR TITLE
Added latitude and longitude to location markers

### PR DESCRIPTION
**Pull request by kevinpark1217**
Displays location in latitude and longitude to all wild Pokemon, gym,
and PokeStops in the marker bubble.
Also, added a gym level indicator.

## Description

It allows easy view of the location coordinates for each location marker. It makes it easy to find and type in the coordinates to any other third party applications.
Ex: GPS Spoofer

## Motivation and Context

It is no longer necessary to press "Inspect Element" to find the location coordinates.

## How Has This Been Tested?

I have tested this on my Ubuntu Server and ran as expected without any problems.
This only adds couple lines to the javascript file; therefore, it was unnecessary to test it more than once. 

## Screenshots:

Pokemon Screenshot
Gym Level Indicator

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by callmebackdraft** [comment]
This is quite a nice addition, however am i the only one who thinks precaution should be taken not to flood the infowindows with alot of info, seeing as they are getting quite big.

**Comment by kevinpark1217** [comment]
@callmebackdraft That is what I was thinking about the gym label because there are a lot of things including the logo. However for wild pokemons, four lines like the screenshot seems reasonable.

I am also considering adding a gym level right next to the gym prestige, so it doesn't take any additional lines.

**Comment by Idhanta** [comment]
+1

**Comment by MonkeyLeeT** [comment]
Tested and working. But ideally it should be off by default (page performance was impacted), can you make some improvement?

**Comment by kevinpark1217** [comment]
@MonkeyLeeT Could you please tell me how much of the page performance was impacted? And how did you test it? I would like to test the page performance myself and find out where the problem is.

I don't know where the performance would be greatly impacted other than the javascript toFixed() method.

**Comment by MonkeyLeeT** [comment]
Oh it was not greatly impacted, it's just http://www.webpagetest.org/ to test some basic metrics. Though small, at this point any changes that might add load to page rendering should be made optional.

**Comment by kevinpark1217** [comment]
@MonkeyLeeT 
I believe that your test is flawed. The test results varied quite heavily depending on what point of time the browser requested for the website. For example, the webserver would take a little longer to respond during a scan rather than a sleep. Therefore, the conclusion is that my changes are too small to be negligible in page rendering process.

I have tested one with my changes first, then the one without my changes. The results are as it follows.
[With Changes](http://www.webpagetest.org/result/160724_8G_72V/)
[Without Changes](http://www.webpagetest.org/result/160724_8J_76Y/)

I have checked multiple times to make sure that I have not mixed up my results.
Hopefully, this resolves the issue.

**Comment by MonkeyLeeT** [comment]
@kevinpark1217 I agree with you. +1 from me.

**Comment by kevinpark1217** [comment]
@MonkeyLeeT So could this be merged with the develop branch?

**Comment by Chlodochar** [comment]
Please match the indentation of the surrounding lines. You're using tabs, the surrounding lines have spaces.

**Comment by MonkeyLeeT** [comment]
@kevinpark1217 please makes changes as suggested above and we'll merge.

**Comment by kevinpark1217** [comment]
@Chlodochar @MonkeyLeeT I have fixed the indentation problem. So, it is ready to merge.

Meanwhile, I have added a gym level indicator, which I have already tested. And it will look like following.
Gym Level Indicator

File: static/map.js [hunk 1]
```diff
@@ -208,19 +212,30 @@ function gymLabel(team_name, team_id, gym_points, latitude, longitude) {
...
             <div>
+                    Location: ${latitude.toFixed(6)}, ${longitude.toFixed(7)}
+            </div>
+            <div>
                 <a href='https://www.google.com/maps/dir/Current+Location/${latitude},${longitude}'
                         target='_blank' title='View in Maps'>Get directions</a>
             </div>
             </center></div>`;
     } else {
+    	var gym_prestige = [2000, 4000, 8000, 12000, 16000, 20000, 30000, 40000$
```
Comment (MonkeyLeeT) [inline static/map.js]: 40000$?

**Comment by MonkeyLeeT** [comment]
Sorry to bother you with one more comments. This should be last.

**Comment by kevinpark1217** [comment]
@MonkeyLeeT Okay. I think I caught them all now. No pun intended.

**Comment by MonkeyLeeT** [comment]
Merged!

**Pull request merged**